Changelog.md

```diff
@@ -1,9 +1,13 @@
 ## 1.1.0 - unreleased
 
+### Added
+
+-   Support for Nordic Thingy:91 X.
```

Comment:
![image](https://github.com/user-attachments/assets/a9768cd2-5d2e-49a5-92ca-8f8b3b66d241)
This needs fixing to the correct naming, with `X`